PhpStorm Protocol (Win)/run_editor.js

```diff
@@ -10,16 +10,41 @@ var settings = {
 };
 
 
-// don't change anything below this line, unless you're know what you're doing
-var url = WScript.Arguments(0),
-	match = /^pstorm:\/\/open\/\?url=file:\/\/(.+)&line=(\d+)$/.exec(url),
-	editor = '"c:\\' + (settings.x64 ? 'Program Files (x86)' : 'Program Files') + '\\JetBrains\\' + settings.folder_name + '\\bin\\PhpStorm.exe" --line %line% "%file%"';
+// don't change anything below this line, unless you know what you're doing
+var url = WScript.Arguments( 0 ),

```

Comment:
Please remove space after `(` and before `)`. Repeat in other places in this PR.